packages/perseus/src/widget-ai-utils/dropdown/dropdown-ai-utils.test.ts

```diff
@@ -59,7 +59,12 @@ describe("Dropdown AI utils", () => {
         };
 
         const renderProps: any = {
-            choices: ["Pickles", "Tomato", "Onion", "Lettuce"],
+            choices: [
+                {content: "Pickles"},
+                {content: "Tomato"},
+                {content: "Onion"},
+                {content: "Lettuce"},
```

Comment:
AI utils basically take the props the widget receives and returns data for a prompt. Since the props are changing from WidgetOptions to RenderProps, the test data also needed to change.